src/main/scala/com/databricks/spark/sql/perf/mllib/data/dataGeneration.scala

```diff
@@ -103,6 +103,30 @@ object DataGenerator {
 
     (sql.createDataFrame(trainPruned), sql.createDataFrame(finalTest))
   }
+
+  def generateRandString(
```

Comment:
Do we need `generateRandString` and `RandStringGenerator`, or could we test everything with `OverlappedStringGenerator` and `generateOverlappedString`? If the latter, then I'd say we should remove `generateRandString` and `RandStringGenerator`.